src/main/scala/nl.knaw.dans.dccd.convert/CommandLineOptions.scala

```diff
@@ -41,12 +41,14 @@ class CommandLineOptions(args: Array[String], configuration: Configuration) exte
        |""".stripMargin)
   //val url = opt[String]("someOption", noshort = true, descr = "Description of the option", default = app.someProperty)
 
-  val runService = new Subcommand("run-service") {
+  val runService = new Subcommand("dccd") {
```

Comment:
runService is misleading, we are not running as a service.

Reply:
I think you meant the value's name here? I agree with that. I already fell for that trap!